src/seedu/addressbook/data/person/Address.java

```diff
@@ -22,6 +26,12 @@
      */
     public Address(String address, boolean isPrivate) throws IllegalValueException {
         String trimmedAddress = address.trim();
+        String[] splitAddress = trimmedAddress.split(",");
```

Comment:
Try to refactor out the delimiter (with a proper name) to make more sense to others.